src/api/IZoweTree.ts

```diff
@@ -180,6 +180,34 @@ export interface IZoweTree<T> extends vscode.TreeDataProvider<T> {
      * Renames a node based on the profile and it's label
      * @deprecated should not be visible outside of class
      *
```

Comment:
The sequence of methods is broken. The method renameNode is disconnected from it's definition

Reply:
Fixed ⚡️